vital/types/detected_object.cxx

```diff
@@ -68,10 +82,27 @@ ::clone() const
   new_obj->m_index = this->m_index;
   new_obj->m_detector_name = this->m_detector_name;
   new_obj->m_descriptor = this->m_descriptor;
+  new_obj->m_geo_point = this->m_geo_point;
 
   return new_obj;
 }
 
+// ------------------------------------------------------------------
+kwiver::vital::geo_point
+detected_object
+::geo_point() const
+{
+  return m_geo_point;
+}
+
+
+// ------------------------------------------------------------------
+void
+detected_object
+::set_geo_point(const kwiver::vital::geo_point& gp)
```

Comment:
If you end up making other changes... [[types.qualified]](../tree/master/doc/code-style.rst) :wink: